### PR TITLE
First modif

### PR DESCRIPTION
**Pull request by TheDiapizator**
To test pull request with ReviewNinja

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/TheDiapizator/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: app/src/main/res/layout/second.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (TheDiapizator) [inline app/src/main/res/layout/second.xml]: Test leave a line comment !resolve  !resolve  !resolved  !resolve

Comment (TheDiapizator) [inline app/src/main/res/layout/second.xml]: !fixed

Comment (TheDiapizator) [inline app/src/main/res/layout/second.xml]: !fix

Comment (TheDiapizator) [inline app/src/main/res/layout/second.xml]: !fixed

**Pull request merged**